build_ngtf.py

```diff
@@ -301,38 +301,9 @@ def main():
     else:
         if arguments.use_prebuilt_tensorflow:
             print("Using existing TensorFlow")
-            # First download the source. This will create the tensorfow directory as needed
```

Comment:
The use_prebuilt_tensorflow mode, was designed to get the tf repo and built the required libraries so that we can run the cpp tests. With tf 1.15.2 due to gcc/abi issues we might not be able to do it temporarily, the idea was to get it back when tf fixes it.

Reply:
And how's that different from not specifying the flag? Based on the usage description [here](https://github.com/tensorflow/ngraph-bridge/blob/master/build_ngtf.py#L98), `use_prebuilt_tensorflow` is supposed to use prebuilt (downloaded or installed) TF instead of building it from source. And because the installed version doesn't distribute the cc libraries, it is supposed to disable CPP tests?

Reply:
yea, i guess the description is not updated.

Reply:
I'm confused — updated to what? If `--use_prebuilt_tensorflow` were to download and build TF then it shouldn't exist in the first place? These changes make the flag do what it says.

Reply:
--use_prebuilt_tensorflow gets TF and builds only certain TF libraries that are required to build and run cpp applications. I think the default way builds a lot more TF libraries. When this mode (--use_prebuilt_tensorflow ) was first added it did not built TF at all, and hence we were not able to run cpp unit tests or cpp applications in examples. Later it was modified to be able to do that and the description might not have been updated. I understand your confusion this mode was extended to do other things one of which was to verify our ability to build and run cpp applications with the gcc/abi pre-built tf was using.

So 2 ways we can proceed:
1. Change the description of the flag
2. Take in your changes. Add another mode (which does what the current mode does) so that when the gcc/abi issues with tf pre-built is resolved we have something to quickly add and test cpp unit tests.

Reply:
What are the ABI issues with prebuilt TF? Why don't we add a mode to be able to just fetch and build `libtensorflow_cc` and run C++ tests against it when these ABI issues are resolved? IMO, `--use_prebuilt_tensorflow` should do what it's supposed to do, and it's incredibly handy to have it.